index.html

```diff
@@ -1 +1,2 @@
-first
\ No newline at end of file
+first
+second feature
```

Comment:
Snygg lösning. Hur kom du på det här?

Reply:
Tack! Jag frågade Gustaf som kom på lösningen!

Reply:
Aha, för att jag brukar lösa det på det här sättet
```suggestion
Second feature
```

Ser du att jag börjar med en stor bokstav. Men känner att din lösning är bättre. Jag godkänner